test/common.bash

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+# bats setup function
+setup() {
+  export KUBECONFIG=$(mktemp)
+  export XDG_CACHE_HOME="$(mktemp -d)"
+}
+
+# bats teardown function
+teardown() {
+  rm -f $KUBECONFIG
+  rm -f $XDG_CACHE_HOME/kubectx
+  rmdir $XDG_CACHE_HOME
+}
+
+# wrappers around "kubectl config" command
+
+add_cluster() {
+  kubectl config set-cluster ${1}
```

Comment:
consider putting "" around all variable references, this prevents splitting by space.